### ISSUE TITLE
sk_region_check_flag returning false when it should be true

### ISSUE THREAD

**Issue by PhanaticD** [jb-aero/SKCompat#28]
If you have a region with the flag pvp -g non_members true it returns false even though they would have pvp

**Comment by PhanaticD** [comment]
this happens if there is two regions, one with a lower priority with pvp deny, and one with a higher priority of pvp allow, the priority must be being ignored or something

**Comment by jb-aero** [comment]
What is the behavior at the overlap of two same-priority regions with different values?

**Comment by PseudoKnight** [comment]
Can't reproduce. I set two overlapping regions with pvp flags set. Lower priority region had pvp deny. Higher priority region had pvp allow. Tried with and without non_members group, with and without player argument of regular non member player. The only way I could produce this result is in ways WorldGuard behaves as expected, like adding the player to the higher priority region when using the non_members group flag. So maybe something has been changed in SKCompat or WorldGuard, or there are more conditions necessary, like parent regions. So closing this as no longer relevant or incomplete.
